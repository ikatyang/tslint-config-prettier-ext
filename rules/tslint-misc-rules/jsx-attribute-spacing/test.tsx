const fails = (
<div>
    <span className = 'bar' />
    <span className= 'bar' />
    <span className ='bar' />
    <span className = { value } />
    <span className= { value } />
    <span className ={ value } />
</div>
);
