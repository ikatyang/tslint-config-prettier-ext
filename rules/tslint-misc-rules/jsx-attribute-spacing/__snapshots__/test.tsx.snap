// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should affect error message after formatting 1`] = `
"
<<<<<< before
ERROR: (jsx-attribute-spacing) /rules/tslint-misc-rules/jsx-attribute-spacing/test.tsx[3, 11]: jsx attribute 'className' should not have whitespace around '='
ERROR: (jsx-attribute-spacing) /rules/tslint-misc-rules/jsx-attribute-spacing/test.tsx[4, 11]: jsx attribute 'className' should not have whitespace around '='
ERROR: (jsx-attribute-spacing) /rules/tslint-misc-rules/jsx-attribute-spacing/test.tsx[5, 11]: jsx attribute 'className' should not have whitespace around '='
ERROR: (jsx-attribute-spacing) /rules/tslint-misc-rules/jsx-attribute-spacing/test.tsx[6, 11]: jsx attribute 'className' should not have whitespace around '='
ERROR: (jsx-attribute-spacing) /rules/tslint-misc-rules/jsx-attribute-spacing/test.tsx[7, 11]: jsx attribute 'className' should not have whitespace around '='
ERROR: (jsx-attribute-spacing) /rules/tslint-misc-rules/jsx-attribute-spacing/test.tsx[8, 11]: jsx attribute 'className' should not have whitespace around '='

======
no error

>>>>>> after
"
`;

exports[`should be pretty after formatting 1`] = `
"
<<<<<< before
const fails = (
<div>
    <span className = 'bar' />
    <span className= 'bar' />
    <span className ='bar' />
    <span className = { value } />
    <span className= { value } />
    <span className ={ value } />
</div>
);

======
const fails = (
  <div>
    <span className=\\"bar\\" />
    <span className=\\"bar\\" />
    <span className=\\"bar\\" />
    <span className={value} />
    <span className={value} />
    <span className={value} />
  </div>
);

>>>>>> after
"
`;
